citadel/install_windows.md

```diff
@@ -0,0 +1,73 @@
+# Binary Installation on Windows 10
+
+Only the following Ignition packages are currently available on `conda-forge`,
+the package manager Ignition uses for Windows binary installations:
+- `ign-common`
+- `ign-msgs`
+- `ign-math`
+- `ign-cmake`
+- `ign-fuel-tools`
+- `ign-transport`
+- `ign-tools`
+- `ign-physics`
+- `ign-plugin`
+
+The remaining packages currently have an [outstanding ticket](https://github.com/conda-forge/staged-recipes/issues/13551),
+and will not be supported until they are added to the `conda-forge` feedstock.
+
+Additionally, command line tools, the DART physics engine, and tests are not currently supported in Windows.
+
+The above mentioned Ignition packages are available in Windows 10 using the [conda-forge package manager](https://conda-forge.org/),
+and the Ignition feedstock recipes can be found [here](https://github.com/conda-forge?q=libignition&type=&language=).
```

Comment:
Can we move this little intro to conda to the top of the tutorial. Right now it's starting a bit abruptly

Reply:
72f9eba4f94965dd2bfc9b3a1b778344953a5594